app/src/main/java/ch/epfl/sweng/swenggolf/database/FakeDatabase.java

```diff
@@ -24,7 +27,7 @@
     /**
      * Create a new FakeDatabase that can be used to mock the Database.
      *
-     * @param working if the database work or send errors
+     * @param working working state of the Database
```

Comment:
I find it's not clear what means the "working state" if I just read this class and don't know what it is doing. You could add some precision, like saying what happens if we make a request and the working parameter was set to false.